submissions/ivan-chukhalo/side-menu/index.js

```diff
@@ -0,0 +1,122 @@
+    description:
+    "У бою шаман ставить на землю тотеми, що контролюють і завдають шкоди, щоб допомогти союзникам і послабити противника. Шамани можуть вступати в ближній бій, так і атакувати з відстані. Мудрий шаман завжди намагається враховувати сильні та слабкі сторони ворога.",
+  },
+  {
+    name: "Маг",
+    description:
+    "Маги знищують ворогів таємними заклинаннями. Незважаючи на магічну силу, маги крихкі, не носять важких обладунків, тому вразливі у ближньому бою. Розумні маги за допомогою заклинань утримують ворога на відстані або зовсім знерухомлюють його.",
+  },
+  {
+    name: "Чорнокнижник",
+    description:
+    "Чорнокнижники знищують ослабленого противника, поєднуючи хвороби і темну магію. Перебуваючи під захистом своїх вихованців, чаклуни розбивають ворога на відстані. Фізично слабкі чаклуни що неспроможні носити важку броню, тому підставляють під ворожі удари своїх слуг.",
+  },
+  {
+    name: "Чернець",
+    description:
+      "Незалежно від стилю ченці в бою покладаються в основному на свої кулаки і ноги, а зв'язок з внутрішньою енергією ци надає їм сил для використання особливих здібностей. Вони здатні одночасно зцілювати союзників і завдавати шкоди противникам.",
+  },
+  {
+    name: "Друїд",
+    description:
+    "Друїди можуть підходити до битви по-різному. Вони вільні грати майже будь-яку роль у команді: бути цілителями, танками чи бійцями, але повинні пам'ятати про особливості кожної ролі. Друїд змушений уважно підбирати вигляд до ситуації, оскільки кожен із навичків служить певної мети.",
+  },
+  {
+    name: "Мисливець на демонів",
+    description:
+      "Мисливці на демонів не користуються важкою бронею, натомість покладаючись на швидкість, яка дає їм можливість стрімко наближатися до супротивника і завдавати йому шкоди одноручною зброєю. Однак іллідары не варто забувати про те, щоб використовувати свою спритність і в захисних цілях, забезпечуючи собі сприятливий результат битви.",
+    },
+    {
+      name: "Лицар смерті",
+      description:
+      "Лицарі смерті сходяться з супротивником у ближньому бою, доповнюючи удари клинка темною магією, яка робить ворога вразливим або ранить його безбожною енергією. Вони провокують противників, змушуючи їх боротися віч-на-віч і не підпускаючи їх до слабших союзників. Щоб не дати супротивникові вислизнути, лицарі смерті повинні постійно пам'ятати про силу, яку витягують з рун, і відповідним чином спрямовувати свої атаки.",
+    },
+  ];
+  
+  function printClassInformation(event) {
+    const contentTitle = document.querySelector('.content__title');
+    const contentDescription = document.querySelector('.content__description');
+    const contentImg = document.querySelector('.content__img');
+    const contentLinkHome = document.querySelector('.content__link_back');
+
+    const menuItem = event.target.closest("a");
+    const className = menuItem.innerText;
+    const classIndex = classesArr.findIndex((el) => {
+    if (el.name === className) {
```

Comment:
Use [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset). And compare id field from data with target.dataset.id

Reply:
I made requested changes. Hope I understood you correctly.